modules/ROOT/pages/proc_installing_cli.adoc

```diff
@@ -0,0 +1,49 @@
+= Installing the CLI
+
+In order to interact with Tekton from a terminal, you need to install the Tekton CLI first.
```

Comment:
Since we use `tkn` throughout this document, I suggest we add `tkn` in line #3: 
...install the Tekton CLI (`tkn`).
the word 'first' is superfluous here.